### PR TITLE
Support Option type in Contract Tab

### PR DESCRIPTION
**Pull request by BriungRi**

### CODE REVIEW

**Review by kaw2k — APPROVED** [review]
Love it, just tested with `run` and `view`.

File: src/pages/Account/Tabs/ModulesTab/Contract.tsx [hunk 1]
```diff
@@ -683,16 +688,18 @@ function ContractForm({
                 <TextField label="signer" disabled fullWidth />
               ))}
             {fnParams.map((param, i) => {
+              // TODO: Need a nice way to differentiate between option and empty string
```
Comment (kaw2k) [inline src/pages/Account/Tabs/ModulesTab/Contract.tsx:691]: Talking with @BriungRi , we would probably want a dropdown between some and none

**Comment by kaw2k** [comment]
@kent-white thoughts?

**Review by kent-white — APPROVED** [review]
looks good to me.

**Pull request merged**